src/message/modalComponent.js

```diff
@@ -123,7 +131,9 @@ class Controller {
     this.resizable = !!this.resizable;
 
     const dialog = this.modal_.find('.modal-dialog');
-    dialog.draggable();
+    dialog.draggable({
+      handle: this.draggableHandle || '.modal-header'
```

Comment:
To allow clicking on the close buttons, shouldn't the default be `.modal-title`?

Reply:
The `.modal-title` is not always present, so what you implemented is better.